proto/rpc/v1/tx.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package rpc.v1;
+
+import "rpc/v1/transaction.proto";
+import "rpc/v1/meta.proto";
+
+message TxRequest {
+    // the hash of the transaction
+    bytes hash = 1;
+
+    // if true, return data in binary format
+    // TODO: does this make sense for protobuf?
+    bool binary = 2;
+}
+
+message TxResponse {
+
+    // The actual transaction
+    oneof sttx {
+        Transaction tx = 1;
+        bytes tx_bytes = 2;
+    };
+    // Sequence number of ledger that contains this transaction
+    uint64 ledger_index = 3;
```

Comment:
Good catch. I am wondering whether it makes sense to define a separate type for this, so that way one does not have to remember that ledger sequence numbers are uint32

Reply:
changed to uint32